pkg/cmd/version/version.go

```diff
@@ -0,0 +1,35 @@
+package version
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	etcdversion "github.com/coreos/etcd/version"
+
+	kubeversion "k8s.io/kubernetes/pkg/version"
+
+	"github.com/openshift/origin/pkg/version"
```

Comment:
This dependency is incorrect as we start splitting these out of tree.  Update it to take a version interface and force the caller to provide it.  Take inspiration from the version endpoint in `k8s.io/apiserver`, but do not directly reference it.